core/variables.tf

```diff
@@ -41,3 +71,40 @@ variable "rds_password" {
   type        = string
   sensitive   = true
 }
+
+variable "bastion_instance_type" {
```

Comment:
these bastions variables can be moved to `locals`, there is no need for them to be configurable through TFC

Reply:
Updated in [e748908](https://github.com/nvminhtue/tyson_devops_ic/pull/33/commits/e74890870c8896447d4bbd848eb6a984847faa76)